recipes/libint2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libint" %}
+{% set version = "2.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/evaleev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tgz
+  sha256: 253fa3301e6742f5aac9e96dbe2a5707fc73abd6f4a214a8d44f03e113e69f00
+
+build:
+  number: 0
+  binary_relocation: true
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - boost-cpp
+    - eigen
+    - gmp
+    - mpfr
+      
+test:
+  commands:
+    # Verify library
+    - test -f "${PREFIX}/lib/libint2${SHLIB_EXT}"
+    # Inspect linkage
+    - ldd -r -u "${PREFIX}/lib/libint2${SHLIB_EXT}" && return 0  # [linux]
```

Comment:
Just out of curiosity, does the `&& return 0` part do anything here?

Reply:
I don't know why the `return 0`  zero is needed here, I took this line from another recipe

Reply:
In that case I think you can remove it.

Reply:
done